compute/instances/how-to/snapshot-import-export-feature.mdx

```diff
@@ -0,0 +1,89 @@
+---
+meta:
+  title: Using the snapshot import/export feature from the Scaleway console
+  description: This page explains how to import/export Instances snapshots using the Scaleway console
+content:
+  h1: Using the snapshot import/export feature with the API
+  paragraph: This page explains how to import/export Instances snapshots using the Scaleway console
+dates:
+  validation: 2022-07-27
+  posted: 2021-07-27
+---
+
+The snapshot transfer feature allows to export or import Instances snapshots. Snapshots are created using the QCOW2 format and stored in a Scaleway Object Storage Bucket that holds the data.
+
+Any snapshot (a valid QCOW2 image) imported to a Scaleway Object Storage bucket, can be imported into any Availability Zone in the same region as the bucket. This allows you to create Instances in other availability zones within the same region, for example you can deploy Instances in `fr-par-2` based on the snapshot exported from an Instance in `fr-par-1`.
+
+
+More information on this file format, and how to use it can be found in the [QEMU documentation](https://www.qemu.org/docs/master/system/images.html#cmdoption-image-formats-arg-qcow2).
```

Comment:
```suggestion
More information on the QCOW2 file format, and how to use it can be found in the [QEMU documentation](https://www.qemu.org/docs/master/system/images.html#cmdoption-image-formats-arg-qcow2).
```